crates/torii/grpc/src/server/mod.rs

```diff
@@ -87,6 +89,29 @@ impl From<SchemaError> for Error {
     }
 }
 
+impl From<Token> for proto::types::Token {
+    fn from(value: Token) -> Self {
+        Self {
+            contract_address: value.contract_address,
+            name: value.name,
+            symbol: value.symbol,
+            decimals: value.decimals as u32,
+            metadata: serde_json::to_string(&value.metadata).unwrap(),
```

Comment:
We may have an expect here or it's just unlikely to happen?

Reply:
We don't even need the serde_json::to_string sorry. You can discard that